### PR TITLE
=doc #20885 small fix and clarification for Java stream-graphs example

### PR DESCRIPTION
**Pull request by gosubpl**

### CODE REVIEW

**Comment by gosubpl** [comment]
Refs #20885

**Comment by ktoso** [comment]
OK TO TEST

File: akka-docs/rst/java/code/docs/stream/GraphDSLDocTest.java [hunk 1]
```diff
@@ -50,25 +50,24 @@ public void demonstrateBuildSimpleGraph() throws Exception {
     //#simple-graph-dsl
     final Source<Integer, NotUsed> in = Source.from(Arrays.asList(1, 2, 3, 4, 5));
     final Sink<List<String>, CompletionStage<List<String>>> sink = Sink.head();
-    final Sink<List<Integer>, CompletionStage<List<Integer>>> sink2 = Sink.head();
```
Comment (ktoso) [inline akka-docs/rst/java/code/docs/stream/GraphDSLDocTest.java]: thanks :)

File: akka-docs/rst/java/code/docs/stream/GraphDSLDocTest.java [hunk 2]
```diff
@@ -50,25 +50,24 @@ public void demonstrateBuildSimpleGraph() throws Exception {
...
 
     final RunnableGraph<CompletionStage<List<String>>> result =
-      RunnableGraph.<CompletionStage<List<String>>>fromGraph(
+      RunnableGraph.fromGraph(
         GraphDSL
           .create(
-            sink,
-            (builder, out) -> {
+            sink,                // previously created sink
+            (builder, out) -> {  // variables: builder (GraphDSL.Builder) and out (SinkShape)
```
Comment (ktoso) [inline akka-docs/rst/java/code/docs/stream/GraphDSLDocTest.java]: hm, maybe rather "out is the `sink`s out port (we need to use its shape below)"?

**Comment by gosubpl** [comment]
Does it look better (i.e. clearer) now?

**Comment by ktoso** [comment]
Yes, i think this is clearer, thanks!

**Pull request merged**